src/components/LowerRightControls.tsx

```diff
@@ -26,6 +39,19 @@ export function LowerRightControls({
     <section className="fixed bottom-2 right-2 flex flex-col items-end gap-3 pointer-events-none">
       {children}
       <menu className="flex items-center justify-end gap-3 pointer-events-auto">
+        {IS_NIGHTLY_OR_DEBUG && (
+          <Popover className="relative">
+            <Popover.Button className="p-0">
+              <BillingRemaining
+                mode={BillingRemainingMode.ProgressBarFixed}
+                billingActor={billingActor}
+              />
+            </Popover.Button>
+            <Popover.Panel className="absolute right-0 left-auto bottom-full mb-1 w-64 flex flex-col gap-1 align-stretch bg-chalkboard-10 dark:bg-chalkboard-90 rounded shadow-lg border border-solid border-chalkboard-20/50 dark:border-chalkboard-80/50 text-sm">
```

Comment:
```suggestion
            <Popover.Panel className="absolute right-0 left-auto bottom-full mb-1 w-64 flex flex-col gap-1 align-stretch rounded-lg shadow-lg text-sm">
```

We don't need a border on this popover, and I just made the border-radius match up with the inner component while removing the bg so the mismatch doesn't matter.